### PR TITLE
Star-347: Update Cura Connect models

### PR DESCRIPTION
**Pull request by marijnd**
These are the changes that should be made to the models for cloud printing after cura connect is updated to send the extra fields to the cloud

### CODE REVIEW

File: plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrintJobStatus.py [hunk 1]
```diff
@@ -77,11 +102,28 @@ def _createConfigurationModel(self) -> ConfigurationModel:
...
         model.updateOwner(self.owner)
         model.updateState(self.status)
+        model.setCompatibleMachineFamilies(self.compatible_machine_families)
+        model.updateTimeTotal(self.time_total)
+        model.updateTimeElapsed(self.time_elapsed)
+        model.updateOwner(self.owner)
+
+        status_set_by_impediment = False
+        for impediment in self.impediments_to_printing:
+            if impediment.severity == "UNFIXABLE":  # TODO: impediment.severity is defined as int, this will not work, is there a translation?
```
Comment (ChrisTerBeke) [inline plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrintJobStatus.py]: Regarding the TODO: the ClusterUM3OutputDevice probably has logic for this?

File: plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrintJobStatus.py [hunk 2]
```diff
@@ -4,11 +4,14 @@
 
 from cura.PrinterOutput.ConfigurationModel import ConfigurationModel
 from plugins.UM3NetworkPrinting.src.Cloud.CloudOutputController import CloudOutputController
-from .CloudClusterPrinterConfiguration import CloudClusterPrinterConfiguration
+from src.ConfigurationChangeModel import ConfigurationChangeModel
```
Comment (ChrisTerBeke) [inline plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrintJobStatus.py]: non-relative import

File: plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrintJobStatus.py [hunk 3]
```diff
@@ -77,11 +102,28 @@ def _createConfigurationModel(self) -> ConfigurationModel:
...
+        for impediment in self.impediments_to_printing:
+            if impediment.severity == "UNFIXABLE":  # TODO: impediment.severity is defined as int, this will not work, is there a translation?
+                status_set_by_impediment = True
+                model.updateState("error")
+                break
+
+        if not status_set_by_impediment:
+            model.updateState(self.status)
+
+        model.updateConfigurationChanges([ConfigurationChangeModel(type_of_change=change.type_of_change,
```
Comment (ChrisTerBeke) [inline plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrintJobStatus.py]: This indentation is quite hard to read.

File: plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrintJobStatus.py (hunk 1 above)
Comment (marijnd) [reply plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterPrintJobStatus.py]: No it doesn't, I checked, it  expects a string

**Pull request merged**